changelog/8456.bugfix.rst

```diff
@@ -0,0 +1 @@
+The `required_plugins` config option now works correctly when pre-releases of plugins are installed, rather than falsely claiming that those plugins aren't installed at all.
```

Comment:
```suggestion
The :confval:`required_plugins` config option now works correctly when pre-releases of plugins are installed, rather than falsely claiming that those plugins aren't installed at all.
```